app/forms/mobile_number_form.rb

```diff
@@ -15,7 +15,6 @@ class MobileNumberForm < Form
 
   def save
     return false unless valid?
-    return true unless mobile_number_changed?
```

Comment:
I'm pretty sure this line meant that the 'resend passcode' functionality never worked.
Also, I can delete this line, and the spec still passes with no changes.